### ISSUE TITLE
add server configure command: RAM and CPU options only

### ISSUE THREAD

**Issue by OriHoch** [cloudwm/cloudcli#9]
```
configure					Change server configuration.

			--cpu {see --list-options) 		CPU count and type.

			--ram {see --list-options) 		Size of server ram in MiB.
```

**Comment by OriHoch** [comment]
remaining options will be added in #22

**Comment by OriHoch** [comment]
implemented in [cloudcli-server v0.1.5](https://github.com/cloudwm/cloudcli-server/releases)
